temporalcli/commands.activity_test.go

```diff
@@ -106,6 +106,32 @@ func (s *SharedServerSuite) TestActivity_Fail_InvalidDetail() {
 	s.Nil(failed)
 }
 
+func (s *SharedServerSuite) TestActivityOptionsUpdate_Accept() {
```

Comment:
Can we add one more test confirming a partial update doesn't mutate the existing value?

Reply:
what are we testing with that? If server implementation - it is already tested.

Reply:
Just an extra confirmation. Many (most?) of our CLI tests are end to end tests with the server implementation. We don't have to test everything, but I would like to confirm that only setting some CLI flags leaves other values intact.

Reply:
ok